FSJ_django20_project/FSJ/templates/FSJ/coord_student_list.html

```diff
@@ -0,0 +1,53 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %}
+
+{% block heading %}
+    Students
+{% endblock %}
+
+{% block content %}
+<head>
+<style>
```

Comment:
Style should be moved to the applicable css file to keep the template clean and also makes style formatting for the entire app much simpler.